### PR TITLE
Replace existing UI with barebones setup based on Momentum's UI

### PR DESCRIPTION
**Pull request by JJL772**
As we discussed... scrapping the entirety of the existing stuff and starting from scratch based on Momentum's work.

I've ripped out everything that's Momentum specific except for the line graph panel because that could be useful for challenge mode stuff one day (plus it's JS/XML-only, so no C++ backend).

Probably missed some things, feel free to clean this up further!

Only big thing missing here is translations. We'll probably want to commit a basic English-only localizations file while we're still fleshing out the UI. Once we're near the final state, we can switch to poeditor or a similar service.

### CODE REVIEW

**Comment by ozxybox** [comment]
Do we have LFS on for these binary files?

**Comment by JJL772** [comment]
We don't

**Comment by ozxybox** [comment]
Should we?

**Comment by JJL772** [comment]
> Should we?

I'm not really sure that using LFS on github is a good idea. I think people simply cloning this repo will contribute to our LFS data transfer limits.

Only ended up including a png for the logo here because we don't have a panorama-compatible SVG yet and I didn't feel like making one :P 
In general, most of the files in here are quite small, so I don't think LFS is really needed. (How often are we going to be updating fonts, anyway?)

**Comment by JJL772** [comment]
Ok so just a note, I (or somebody else) needs to do the following things to this PR:
* Change font to the DIN font we use
* Provide a temp localization file for the strings used in here

File: tools/get_latest_js_types.bat [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@echo off
+:: Get latest types file and plop it in scripts
+
+powershell Invoke-WebRequest https://github.com/panorama-languages-support/panorama-jsdoc-gen/releases/latest/download/types_momentum.zip -OutFile types.zip
```
Comment (craftablescience) [inline tools/get_latest_js_types.bat:4]: We're not momentum, I'm pretty sure this generates types for p2ce too

File: tools/get_latest_js_types.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Get latest types file and cram it into scripts
+
+wget "https://github.com/panorama-languages-support/panorama-jsdoc-gen/releases/latest/download/types_momentum.zip"
```
Comment (craftablescience) [inline tools/get_latest_js_types.sh:4]: ditto

File: fonts/FontCredits.txt [hunk 3]
```diff
@@ -0,0 +1,512 @@
+Momentum uses the following fonts with their respective licenses below:
```
Comment (craftablescience) [inline fonts/FontCredits.txt:1]: Not momentum

**Review by craftablescience — CHANGES_REQUESTED** [review]

File: tools/get_latest_js_types.bat (hunk 1 above)
Comment (AWildErin) [reply tools/get_latest_js_types.bat:4]: This isn't needed for us, it's inside sdk_tools

File: .github/workflows/format.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+name: Formatting
+
+on:
+  pull_request:
+    branches: [ master ]
```
Comment (craftablescience) [inline .github/workflows/format.yml:5]:
```suggestion
    branches: [ main ]
```

File: dev_keybinds.cfg [hunk 5]
```diff
@@ -5,5 +5,4 @@ key_f5 Refresh
 key_f6 ToggleDebugger
 key_f7 ReloadPanorama
 key_f8 ForceReloadPanorama
-key_f9 ShowPanelZoo
-
+key_f9 ShowPanelZoo
```
Comment (craftablescience) [inline dev_keybinds.cfg:8]: 😢

**Review by craftablescience — CHANGES_REQUESTED** [review]
looks fine ingame, but we might want a warning that its heavily wip or smth to that effect before it ships

File: dev_keybinds.cfg (hunk 5 above)
Comment (JJL772) [reply dev_keybinds.cfg:8]: ?

**Comment by JJL772** [comment]
I'll fix the workflow here shortly

File: .github/workflows/format.yml (hunk 4 above)
Comment (JJL772) [reply .github/workflows/format.yml]: Done

**Comment by JJL772** [comment]
image

**Review by craftablescience — APPROVED** [review]

**Pull request merged**